jobs/scheduler/spec

```diff
@@ -57,6 +60,14 @@ properties:
     default: 3
     description: "Maximum number of notification sent to scaling engine for job re-schedule"
 
+  autoscaler.cf_server.xfcc.valid_org_guid:
```

Comment:
I would following the convention already:
e.g.,
autoscaler.scheduler.cf_server.xfcc.valid_org_guid

Reply:
good idea. Fixed 949502985